prometheus/prometheus.sh

```diff
@@ -28,7 +28,7 @@ readonly KAFKA_CONFIG_FILE=${KAFKA_CONFIG_DIR}/server.properties
 readonly KAFKA_LIBS_DIR=/usr/lib/kafka/libs
 readonly KAFKA_JMX_JAVAAGENT_VERSION='0.11.0'
 readonly KAFKA_JMX_JAVAAGENT_NAME="jmx_prometheus_javaagent-${KAFKA_JMX_JAVAAGENT_VERSION}.jar"
-readonly KAFKA_JMX_JAVAAGENT_URI="http://central.maven.org/maven2/io/prometheus/jmx/jmx_prometheus_javaagent/${KAFKA_JMX_JAVAAGENT_VERSION}/${KAFKA_JMX_JAVAAGENT_NAME}"
+readonly KAFKA_JMX_JAVAAGENT_URI="https://repo1.maven.org/maven2/io/prometheus/jmx/jmx_prometheus_javaagent/${KAFKA_JMX_JAVAAGENT_VERSION}/${KAFKA_JMX_JAVAAGENT_NAME}"
```

Comment:
@medb I tested this script using the set-up listed here. Although the packages on this line are downloaded as expected, there was an error that said "kafka-server-start.sh" not found (line 134 in this file).

I tried adding kafka in initialization-actions but that did led to some issues with Zookeeper.

Since I have not used prometheus before, could you tell me if I am missing any set-up here?

Reply:
I think that you need to create cluster with Kafka init action to test Prometheus integration w/ Kafka:
https://github.com/GoogleCloudDataproc/initialization-actions/tree/master/kafka

Note, that you need to specify Kafka init action before Prometheus init action.

Reply:
I already tried that, but as mentioned it led to issues with zookeeper. Did zookeeper have to be included too?

Also can we update the init action steps listed [here](https://github.com/GoogleCloudDataproc/initialization-actions/tree/master/prometheus), since they do not work as is?

Reply:
Exception in thread "main" org.apache.zookeeper.KeeperException$ConnectionLossException: KeeperErrorCode = ConnectionLoss for /
	at org.apache.zookeeper.KeeperException.create(KeeperException.java:102)
	at org.apache.zookeeper.KeeperException.create(KeeperException.java:54)
	at org.apache.zookeeper.ZooKeeper.getChildren(ZooKeeper.java:1541)
	at org.apache.zookeeper.ZooKeeper.getChildren(ZooKeeper.java:1569)
	at org.apache.zookeeper.ZooKeeperMain.processZKCmd(ZooKeeperMain.java:732)
	at org.apache.zookeeper.ZooKeeperMain.processCmd(ZooKeeperMain.java:600)
	at org.apache.zookeeper.ZooKeeperMain.run(ZooKeeperMain.java:363)
	at org.apache.zookeeper.ZooKeeperMain.main(ZooKeeperMain.java:291)
+ echo 'Failed to connect to ZooKeeper prometheus-3-m:2181, retry 10...'

Reply:
Resolved the issue by creating HA cluster. 

Maybe we should update init-actions instructions to avoid confusion?